app/controllers/live.js

```diff
@@ -36,6 +37,10 @@ export default class LiveController extends Controller {
     }, 4000);
   }
 
+  @action sidebarPanel() {
```

Comment:
sidebarPanel doesn't tell me what action it is performing, please be more accurate towards the action you're doing here.
Like sidebarDisplayToggle can be used here.